src/cpu/o3/fetch.cc

```diff
@@ -835,7 +835,7 @@ Fetch::finishTranslation(const Fault &fault, const RequestPtr &mem_req)
         // If we have, just wait around for commit to squash something and put
         // us on the right track
         if (!cpu->system->isMemAddr(mem_req->getPaddr())) {
-            warn("Address %#x is outside of physical memory, stopping fetch, %lu\n",
+            DPRINTF(Fetch, "Address %#x is outside of physical memory, stopping fetch, %lu\n",
```

Comment:
Use "warn" can help us find the fetch stuck easier

Reply:
> Use "warn" can help us find the fetch stuck easier

But if on the speculative path, might it be too verbose?